packages/node_modules/@webex/internal-plugin-conversation/test/unit/spec/conversation.js

```diff
@@ -104,5 +104,16 @@ describe('plugin-conversation', () => {
         assert.equal(url, convoUrl);
       });
     });
+
+    describe('paginate', () => {
+      it('should throw an error if a page is passed with no links', () => {
+        try {
```

Comment:
You are getting test failures with this. Let's use the `isRejected` helper:

```suggestion
        assert.isRejected(webex.internal.conversation.paginate({page: {}}), 'No link to follow for the provided page');
```

Reply:
Sure. But I copied the same error message handling from another test. I wonder if it's how I'm rejecting. I'll take a look. Thanks!

Reply:
It was the return statements. Pushing a fix now